scap-workbench-oscap.sh

```diff
@@ -74,7 +74,7 @@ while kill -0 $PID 2> /dev/null; do
     ret=$?
     if [ 0 -lt $ret -a $ret -lt 128 ]; then
         # If read failed & it was not due to timeout --> parents are gone.
-        kill -s SIGINT $PID 2> /dev/null
+        kill -TERM $PID 2> /dev/null
```

Comment:
Well, I'm really not an expert on which signal to use here, however, SIGINT is not an option as it does not reach the oscap tool.

What signal do you think that fits better?